go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/libp2p/go-libp2p
 go 1.12
 
 require (
-	github.com/gogo/protobuf v1.3.2
+	github.com/gogo/protobuf v1.3.1
+	github.com/google/gopacket v1.1.18 // indirect
+	github.com/google/uuid v1.1.2 // indirect
```

Comment:
what's that crap popping up? Do a `go mod tidy`.

Reply:
Even a go mod tidy dosen't get rid of it.

```
aarshshah@Aarshs-MacBook-Pro go-libp2p % go mod why -m github.com/google/uuid
# github.com/google/uuid
github.com/libp2p/go-libp2p/p2p/host/routed
github.com/libp2p/go-libp2p-loggables
github.com/google/uuid
aarshshah@Aarshs-MacBook-Pro go-libp2p % go mod why -m github.com/google/gopacket
# github.com/google/gopacket
github.com/libp2p/go-libp2p/p2p/host/basic
github.com/libp2p/go-netroute
github.com/google/gopacket/routing
```